src/bun.js/modules/Buffer.h

```diff
@@ -0,0 +1,27 @@
+#include "../bindings/ZigGlobalObject.h"
```

Comment:
Can we use unique file names? It will make it easier to CMD + F in project

Instead of `Buffer.h`, `BufferModule.h`

Reply:
fixed.